app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -362,8 +362,33 @@ def parse_container(container, pod_id)
         :backing_ref     => container.containerID,
         :container_image => parse_container_image(container.image, container.imageID)
       }
-
-      # TODO, state
+      Hash(container.state).each do |key, val|
```

Comment:
Yeah, I agree with @matthewd here.  Convert `container.state` to an array to capture the `state` and `state_info`.  Then, you can parse the `state_info` as a simple hash.

You should probably add a comment to the `container.state.to_a` call to make it clear what you're parsing.  In fact, you might just want to pull this all out to a method that parses the container state, and add a comment there with these examples, so it's clear exactly what's getting parsed.